server/schedule/checker/merge_checker.go

```diff
@@ -146,7 +149,11 @@ func (m *MergeChecker) Check(region *core.RegionInfo) []*operator.Operator {
 		return nil
 	}
 
-	if target.GetApproximateSize() > maxTargetRegionSize {
+	maxSize := int64(maxRegionFactor * m.cluster.GetOpts().GetMaxRegionSize())
```

Comment:
Will this affect the duration of calling `Check`?

Reply:
I think it will not affect the performance of checker,  the function of `GetMaxRegionSize()` get constant value only.